CMakeLists.txt

```diff
@@ -14,11 +14,15 @@ include_directories("${CMAKE_SOURCE_DIR}/include/")
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -O0 -Wall")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O2")
-IF(NOT WIN32)
+IF(MSVC)
+  set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -W4")

```

Comment:
Is that syntax accepted? 
I'm only aware of the flags being accepted with a forward slash: /W4